.babelrc

```diff
@@ -1,8 +1,11 @@
 {
   "presets": [
-    "react-app"
```

Comment:
babel needs many plugins to finish code compilation (or transformation) process. A preset is a pre-determined set of plugins and with presets you don't have to add all related plugins one by one.

Here, I modify the presets bacause the upstream of this mv3 boilerplate modified it: https://github.com/lxieyang/chrome-extension-boilerplate-react/commit/03e24851f283a642e0d9dfb63e6f99b7c1ed7f79